cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -182,6 +182,12 @@
   private static final String DEFAULT_POD_LABELS_FILE = "pod.labels.properties";
   private static final long DEFAULT_POD_KILLER_DELAY_MILLIS = TimeUnit.HOURS.toMillis(1L);
 
+  // Environment Property Names
+  private static final String COMPONENT_ENV_PROPERTY =
```

Comment:
nit: add comments explaining what the property is and roughtly how it is being used

Reply:
explain what "cmp" is

Reply:
Added comment explaining choice of name.